tests/test_autocommit.py

```diff
@@ -0,0 +1,11 @@
+import sys, os
+
+def test():
+    "Verify autocommit just before program exits."
+    assert(0 == os.system('PYTHONPATH=. %s tests/autocommit.py' % sys.executable))
```

Comment:
```suggestion
    assert os.system('PYTHONPATH=. %s tests/autocommit.py' % sys.executable) == 0
```

Reply:
I'm trying to apply the above patch to your branch, but it looks like I don't have permissions.

Can you please tick "allow commits from maintainers" in the PR settings? Alternatively, apply the patch yourself.